_sass/module/_header-nav.scss

```diff
@@ -67,4 +81,8 @@
 
 .header-nav__trigger:checked ~ .header-nav__items {
   display: block;
+
+  @media screen and (max-width: 768px) {
```

Comment:
You should use the [sass variable](https://github.com/caciviclab/odca-jekyll/blob/feature/update-caciviclab-url/_sass/layout/_grid.scss#L2) here and the [media mixin](https://github.com/caciviclab/odca-jekyll/blob/386fef6ed165ebc602c0f975b3e0542e21b5123c/_sass/layout/_grid.scss#L34).

Reply:
Hey Aaron! I implemented the sass variable and media mixin (hopefully correctly). Branch passes the tests. You available for final review?